run.go

```diff
@@ -13,9 +13,13 @@ import (
 	"os/exec"
```

Comment:
the diff main.patch is too big. did you create it with the scripts?

Reply:
no, I think I created it wrong. It shouldn't be this big